Formula/allegro.rb

```diff
@@ -15,24 +15,18 @@ class Allegro < Formula
     sha256 "65a04aa3c0901264e54ca91f8982da085ea90bccabcb885fac054ba5219e19bd" => :mavericks
   end
 
-  devel do
-    url "http://download.gna.org/allegro/allegro-unstable/5.1.13.1/allegro-5.1.13.1.tar.gz"
-    sha256 "c6e6265c3d661d46270971b7fbf7db34c1873af62882a9ea4025ca1298edf14d"
-
-    depends_on "theora" => :recommended
-  end
-
   head do
-    url "https://github.com/liballeg/allegro5.git", :branch => "5.1"
-
-    depends_on "theora" => :recommended
+    url "https://github.com/liballeg/allegro5.git", :branch => "5.2"
   end
 
   depends_on "cmake" => :build
   depends_on "libvorbis" => :recommended
   depends_on "freetype" => :recommended
   depends_on "flac" => :recommended
   depends_on "physfs" => :recommended
+  depends_on "dumb" => :recommended
+  depends_on "libogg" => :recommended
+  depends_on "theora" => :recommended
```

Comment:
I'm sorry Dumb is optinal. Allegro's preference is for `libogg` rather than `theora` in 5.2. 
But theora is still repuired.

>$ otooL -L /usr/local/Cellar/allegro52/5.2.0/lib/liballegro_video.dylib 
>/usr/local/Cellar/allegro52/5.2.0/lib/liballegro_video.dylib:
>	/usr/local/opt/allegro52/lib/liballegro_video.5.2.dylib (compatibility version 5.2.0, current version 5.2.0)
>	@loader_path/liballegro_audio.5.2.dylib (compatibility version 5.2.0, current version 5.2.0)
>	/usr/local/Cellar/libtheora/1.1.1/lib/libtheoradec.1.dylib (compatibility version 3.0.0, current version 3.4.0)
>	/usr/local/Cellar/libogg/1.3.2/lib/libogg.0.dylib (compatibility version 9.0.0, current version 9.2.0)
>	/usr/local/Cellar/libvorbis/1.3.5/lib/libvorbisfile.3.dylib (compatibility version 7.0.0, current version 7.7.0)
>	/usr/local/Cellar/libvorbis/1.3.5/lib/libvorbis.0.dylib (compatibility version 5.0.0, current version 5.8.0)
>...